src/main/java/com/patsi/service/LogInSessionService.java

```diff
@@ -40,10 +40,16 @@ public boolean endSession(UUID uid) {
         return false;
     }
 
+    @Transactional
+    public boolean endSessionByToken(String token) {
+        sessionRepository.deleteBySessionToken(token);
+        return true;
+    }
+
     public Person findPersonByToken(String token) {
-        LogInSession s = sessionRepository.findBySessionToken(token);
+        String sessionToken = token.substring(7);
```

Comment:
extract the "Bearer" removing logic to a util/helper

Reply:
Done